README.adoc

```diff
@@ -191,4 +191,4 @@ disabledPlugins: <1>
   - "kotlin.gradle.targets.js" <2>
 ----
 <1> The `disabledPlugins` key (and only this key) must be present followed by the plugin list.
-<2> The list of disabled plugins are strings in tool-specific format, e.g., ``"<pluginId>"` for Gradle.
+<2> The list of disabled plugins are strings in tool-specific format, e.g., ``"<packageNameOfTask>"` for Gradle.
```

Comment:
Should there be a hint on how to find the packageName?

Reply:
I guess you're asking for a way to go from a plugin to all registered tasks to to task types. The task "type" seems to be the name class which implements `org.gradle.api.Task`.

Finding this class will depend on whether it's a bundled Gradle task or an external class. But, how do we even enumerate these classes if we don't know where they're located. For example, how do we know that the `"signing"` plugin has task `org.gradle.plugins.signing.Sign`? And how do we know where to look for it?

Reply:
Thats true. I guess from looking at https://docs.gradle.org/current/userguide/signing_plugin.html there is a link to https://docs.gradle.org/current/javadoc/org/gradle/plugins/signing/SigningExtension.htm and from there we can find https://docs.gradle.org/current/javadoc/org/gradle/plugins/signing/Sign.html 
Of course that linkage isn't always available.